ParisBNF/abb/BNFabb105.xml

```diff
@@ -0,0 +1,979 @@
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ እለ፡ ኖሙ፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አኃዜ፡ ኵሉ፡ አ<pb n="184va"/>ምላክነ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ᎓ አምላከ፡ መንፈስ፡ ወለኵሉ፡ ዘሥጋ፡ በእንተ፡ እለ፡ ኖሙ፡ አኃዊነ፡ ናስተበቍዐከ፡ ከመ፡ ያዕርፍ፡ ነፍሰ፡ ኵሎሙ፡ ውስተ፡ መካን፡ ስዑር፡
+                               </incipit>
+                             </msItem>
+                             <msItem xml:id="ms_i1.6.10">
+                               <locus from="184vb" to="185rb"/>
+                               <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ማኅበር፡</hi></title>
+                               <textLang mainLang="gez"></textLang>
+                                 <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ አምላክነ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                   አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ᎓ በእንተ፡ ማኅበር፡ ናስተበቍዖ፡ ከመ፡ ይትወከፍ፡ ለነ፡ ግብረ፡ ማኅበርነ፡ ንግበር፡
+                               </incipit>
+                               <msItem xml:id="ms_i1.6.11">
+                                 <locus from="185rb" to="185va"/>
+                                 <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ጳጳስነ፡</hi></title>
+                                 <textLang mainLang="gez"></textLang>
+                                   <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                     አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ᎓ በእንተ፡ ብፁዕ፡ ጳጳስነ፡ <hi rend="rubric">እገሌ፡ </hi>ከመ፡ ይትማኅፀኖ፡ ወይዕቀቦ፡
+                                 </incipit>
+                               </msItem>
+
+                               <msItem xml:id="ms_i1.6.12">
+                                 <locus from="185va" to="185vb"/>
+                                 <title type="complete" xml:lang="gez"><hi rend="rubric">ጸሎት፡ በእንተ፡ ሰላም፡</hi></title>
+                                 <textLang mainLang="gez"></textLang>
+                                   <incipit xml:lang="gez">ወከዕበ፡ ናስተበቍዖ፡ ለእግዚአ፡ ኵሉ፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ ዘይነግሥ፡ ለኵሉ፡ ዓለም፡
+                                     አቡሁ፡ ለእግዚእነ፡ ወመድኀኒነ፡ ኢየሱስ፡ ክርስቶስ᎓ ንጉሰ፡ ሰላም፡ በእንተ፡ ሰላም፡ ለቅድስት፡ አኃቲ፡ እንተ፡ ብኵላሃ፡ ቤተ፡
+                                     ክርስቲያን፡
+                                 </incipit>
+                               </msItem>
+                           </msItem>
+                           <msItem xml:id="ms_i1.7">
+                             <locus from="185vb" to="191vb"/>
+                             <title type="complete" xml:lang="gez">Further prayers and readings</title>
+                             <textLang mainLang="gez"/>
+                               <incipit xml:lang="gez"><hi rend="rubric">ካህን፡ ይብል፡ ሰብሐት፡ ለእግዚአብሔር፡ ወሕዝብ፡ ይብሉ፡ ርትዕ᎓᎓</hi> ለከ᎓
```

Comment:
no ref?

Reply:
No. The analysis of this part would take too much time.